src/sentry/static/sentry/app/components/dropdownControl.jsx

```diff
@@ -104,7 +104,6 @@ const StyledDropdownButton = styled(
 )`
   z-index: ${p => p.theme.zIndex.dropdownAutocomplete.actor};
   white-space: nowrap;
-  font-weight: normal;
```

Comment:
Is this what we want? https://percy.io/getsentry/sentry/builds/2055032/view/133452734/1280?browser=firefox&mode=diff

Reply:
Yep! The previous behavior was unintended and this change makes button font-weight consistent.